pkg/api/plugin.go

```diff
@@ -69,48 +69,17 @@ type DeployFn func(context.Context, map[string]interface{}) error
 
 // PluginConfig is passed into NewPlugin
 type PluginConfig struct {
-	SyncImage       string
 	AcceptLanguages []string
```

Comment:
eventually I think this should go in the context

Reply:
metrics and etcbackup too?

Reply:
https://github.com/openshift/openshift-azure/issues/966